From: Arvind Singh <arvind@buzzworks.com>
Date: Sat, 4 May 2013 15:41:08 +0530
Subject: Set perl installation path to vendorlib

---
 xpp/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/xpp/Makefile b/xpp/Makefile
index 6fa04a4..4d61903 100644
--- a/xpp/Makefile
+++ b/xpp/Makefile
@@ -28,7 +28,7 @@ SBINDIR		= $(prefix)/sbin
 DATADIR		= $(datadir)/dahdi
 MANDIR		= $(mandir)/man8
 HOTPLUG_USB_DIR	= $(sysconfdir)/hotplug/usb
-PERLLIBDIR	:= $(shell eval `perl -V:sitelib`; echo "$$sitelib")
+PERLLIBDIR	:= $(shell eval `perl -V:vendorlib`; echo "$$vendorlib")
 PERL_DIRS	:= $(shell cd perl_modules; find * -name '[A-Z]*' -type d| xargs)
 PERL_MODS_PAT	:= *.pm $(PERL_DIRS:%=%/*.pm)
 PERL_MODS	:= $(shell cd perl_modules; echo $(PERL_MODS_PAT))
